packages/web-components/src/global/variables.css

```diff
@@ -191,7 +191,7 @@
   --ic-status-error-dark: #610a05;
   --ic-status-info-light: #e1f0fc;
   --ic-status-info-contrast: #488fe3;
-  --ic-status-info: #124db3;
+  --ic-status-info: rgb(18 77 179);
```

Comment:
this feels unusual - why only change this variable to RGB?

Reply:
My reasoning was because the other status colours (error, warning, etc.) were rgb, so its consistency in that aspect, but also because on line 153 of ic-badge.tsx it could then be converted using the rgbstringtoobj method like the other status colours

Reply:
thanks for explaining - i hadn't looked at the rest of variables.css in a while and didn't realise that we used rgb() for other variables